lms/djangoapps/edxnotes/helpers.py

```diff
@@ -1,35 +1,39 @@
 """
 Helper methods related to EdxNotes.
 """
+
 import json
 import logging
 import requests
+
+from datetime import datetime
```

Comment:
Minor: since you're making changes here, please fix the import order. Requests is a third-party library; datetime, json, and uuid are part of the standard library.